stats/opentelemetry/client_metrics.go

```diff
@@ -150,7 +150,12 @@ func (h *clientStatsHandler) TagRPC(ctx context.Context, info *stats.RPCTagInfo)
 	var labels *istats.Labels
 	if labels = istats.GetLabels(ctx); labels == nil {
 		labels = &istats.Labels{
-			TelemetryLabels: make(map[string]string),
+			// The defaults for all the per call label from a plugin that
```

Comment:
Nits:

"labelS"

"on _the_ call path"

Reply:
Done for both.